src/app/problems/components/problems-table/problems-table.component.html

```diff
@@ -9,7 +9,7 @@
   <!-- Name Column -->
   <ng-container matColumnDef="message">
     <mat-header-cell *matHeaderCellDef> Message </mat-header-cell>
-    <mat-cell *matCellDef="let element"> {{element.message}} </mat-cell>
+    <mat-cell *matCellDef="let element"> {{element.type}} </mat-cell>
```

Comment:
Why type instead of message?

Reply:
That's how data model changed. I think Leszek changed it to match Jonathans data model.